test/e2e/fleetautoscaler_test.go

```diff
@@ -456,72 +456,81 @@ func TestAutoscalerWebhook(t *testing.T) {
 	assert.True(t, found, "Expected error was not received")
 }
 
+// Instructions: https://agones.dev/site/docs/getting-started/create-webhook-fleetautoscaler/#chapter-2-configuring-https-fleetautoscaler-webhook-with-ca-bundle
```

Comment:
fyi @aLekSer , I updated the certs here, and added a comment on how to update and when expiry should happen.